IP Allow List (#3587)
timescale/docs: use-timescale/security/ip-allow-list.md

---
title: IP allow list
excerpt: Create a list of IP addresses that can access your services
product: cloud
keywords: [ip allow list, security]
tags: [ip allow list]
---

# IP allow list

> You can restrict access to your $SERVICE_LONGs on public cloud to trusted IP addresses only. This prevents unauthorized connections without the need for a Virtual Private Cloud. Creating IP allow lists helps comply with security standards, for example, SOC 2 or HIPAA, that require IP filtering, especially in regulated industries like finance, healthcare, and government.

Reviewer: ```suggestion
You can restrict access to your $SERVICE_LONGs from trusted IP addresses only. This prevents unauthorized connections without the need for a Virtual Private Cloud. Creating IP allow lists helps comply with security standards such as SOC 2 or HIPAA that require IP filtering. This is especially useful in regulated industries like finance, healthcare, and government.
```

Reviewer: Can you add a link to the VPC doc where you say Virtual Private Cloud please.

<Highlight type="important">

> You can attach a $SERVICE_SHORT to either one VPC, or one IP allow list. 

Reviewer: ```suggestion
You attach a $SERVICE_SHORT to either one VPC, or one IP allow list. You cannot attach a service to a VPC and an IP allow list at the same time. 
```

</Highlight>

## Create and attach an IP allow list

> You create an IP allow list at the project level, then attach your $SERVICE_SHORT to it.

Reviewer: Link to https://docs.timescale.com/use-timescale/latest/members/ on `project level` please.

<Procedure>

> 1. In [$CONSOLE][console], select `Security` > `IP Allow List` and click `Create IP Allow List`. 

Reviewer: ```suggestion
1. In [$CONSOLE][console], select `Security` > `IP Allow List`, then click `Create IP Allow List`. 
```

   ![Create IP allow list](https://assets.timescale.com/docs/images/create-ip-allow-list.png)

1. Enter your trusted IP addresses. 

   The number of IP addresses that you can include in one list depends on your [pricing plan][pricing-plans]. 

   ![Add IP addresses to allow list](https://assets.timescale.com/docs/images/add-ip-addresses-to-allow-list.png)

1. Name your allow list and click `Create IP Allow List`.

   Click `+ Create IP Allow List` to create another list. The number of IP allow lists you can create depends on your [pricing plan][pricing-plans].

1. Select a $SERVICE_LONG, then click `Operations` > `Security` > `IP Allow List`. 

   ![Attach IP allow list](https://assets.timescale.com/docs/images/attach-ip-allow-list.png)

1. Select the list in the drop-down and click `Apply`. 

1. Type `Apply` in the confirmation popup.

</Procedure>

> 

Reviewer: Add a line explaining what has been done, and that you use the allow list pane to update and delete allow lists bla bla
